tests/MongoDB.Driver.Core.Tests/Core/Operations/FindCommandOperationTests.cs

```diff
@@ -818,6 +821,34 @@ public void CreateCommand_should_throw_when_ReadConcern_is_set_but_not_supported
             result.Should().Be(expectedResult);
         }
 
+        [SkippableFact]
+        public void CreateCursor_should_use_ns_field_instead_of_namespace_passed_in_constructor()
+        {
+            var subject = new FindCommandOperation<BsonDocument>(_collectionNamespace, BsonDocumentSerializer.Instance, _messageEncoderSettings);
+            var expectedCollectionNamespace = "cursors.lkajlkasdf-3980238d908sdf";
+            var nextBatchSlice = new ByteArrayBuffer(new byte[] { 5, 0, 0, 0, 0 }, isReadOnly: true);
+            var cursorDocument = new BsonDocument
+            {
+                { "id", 0 },
+                { "firstBatch", new RawBsonArray(nextBatchSlice) },
+                { "ns", expectedCollectionNamespace }
+            };
+            var mockExecuteResponse = new BsonDocument
+            {
```

Comment:
Add:

    { "ok", 1 }

I know it's not needed for the test but it's a more realistic representation of what the command result looks like.

Reply:
Done.